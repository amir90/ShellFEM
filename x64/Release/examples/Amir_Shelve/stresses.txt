          0
1.11071e+12
1.07522e+12
1.63743e+12
1.88301e+11
1.90134e+11
1.70316e+12
 1.4406e+09
5.03482e+08
1.69978e+12
1.69649e+12
1.03317e+12
1.11651e+12
1.02609e+12
1.15435e+12
          0
1.06799e+12
1.82258e+11
7.71083e+09
2.12185e+11
1.65756e+12
1.93453e+11
1.70761e+12
 9.9169e+11
          0
          0
1.68207e+12
1.82459e+12
1.90552e+11
1.81052e+11
1.86116e+11
3.87565e+09