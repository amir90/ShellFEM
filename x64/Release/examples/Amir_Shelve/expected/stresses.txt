0 675638.6875
1 167685.265625
2 740529.0625
3 184172.90625
4 933224.6875
5 935119.5625
6 0.0
7 0.0
8 0.0
9 0.0
10 933370.3125
11 173171.6875
12 0.0
13 933370.3125
14 0.0
15 0.0
16 740529.0625
17 184172.90625
18 933078.25
19 933370.3125
20 184172.90625
21 167685.265625
22 681115.875
23 681115.875
24 173171.6875
